modules/govuk_jenkins/templates/jobs/scrape_icinga_alerts_for_dashboard_metrics.yaml.erb

```diff
@@ -28,7 +28,7 @@
        - shell: |
           export RBENV_VERSION=2.5.1
```

Comment:
These 2 lines might not be necessary anymore?

Reply:
👍 I've now removed RBENV_VERSION, but keeping the PATH envvar for now as we think Jenkins needs it to be able to find Bundler... but shout if you know better!

Reply:
Not sure, I haven't seen this used in any other Jenkins script?

https://github.com/alphagov/govuk-puppet/search?q=%22bundle+install+--path+%22%24%7BHOME%7D%2Fbundles%2F%24%7BJOB_NAME%7D%22%3B%22&unscoped_q=%22bundle+install+--path+%22%24%7BHOME%7D%2Fbundles%2F%24%7BJOB_NAME%7D%22%3B%22

Reply:
Touché. Removed.